gridstatus/ercot.py

```diff
@@ -555,7 +555,9 @@ def _get_spp_dam(
     ):
         """Get day-ahead hourly Market SPP data for ERCOT"""
         if date == "latest":
-            raise ValueError("DAM is released daily, so use date='today' instead")
+            raise ValueError(
+                "DAM is released daily, so use date='today' instead",
```

Comment:
Sidebar: I'm surprised these formatting changes weren't enforced by isort/black in the pre-commit hook earlier on.

Reply:
hmm. there were some changes in #71 that may be the reason